tests/documentation/DocGrammarBearTest.py

```diff
@@ -0,0 +1,75 @@
+from queue import Queue
+import os.path
+import unittest
+import shutil
+
+from coalib.results.Diff import Diff
+from coalib.settings.Section import Section
+from coalib.testing.LocalBearTestHelper import execute_bear
+
+from bears.documentation.DocGrammarBear import DocGrammarBear
+
+try:
+    import language_check
+    language_check
```

Comment:
also can you use this decorator instead? http://github.com/coala/coala/blob/master/coalib/testing/BearTestHelper.py#L4